### PR TITLE
docs: add info about reporting potential vulnerabilities

### LINKED ISSUE #831: Add information regarding vulerability report
- [ ] Add another line to the Readme like “For any vulnerability reports, please reach out to us at contact@metabolicatlas.org”.
- [ ] Update CII badge https://bestpractices.coreinfrastructure.org/en/projects/4276

### PR DESCRIPTION
**Pull request by e0**
**Related issue(s) and PR(s)**  
This PR closes #831 .

**Type of change**  

- [x] Other
 
**List of changes made**  

Added a line including contact email for reporting vulnerabilities.

**Further comments**  

Reading the issue description for #831, I initially thought this was about us needing to provide a report/document of the project's vulnerabilities upon request. I then read the section on the CII badge website and it sounds like that we just need to provide a contact email for others who want to report a potential vulnerability. It also states that "This MAY be the same as its bug reporting process. ", which is already covered under the `CONTRIBUTING` document. Just wanted to mention this here in case I misread the description.

- [ ] Update CII Badge <- This should be done after the PR is approved and merged.

**Checklist**  

- [x] I have made corresponding changes to the documentation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,6 +13,8 @@ Metabolic Atlas is a web platform integrating open-source genome scale metabolic
 ## Contributing
 If you would like to contribute to the project, for example if you have ideas for new features, discovered a bug or if you would like to improve the code base, please have a look at [CONTRIBUTING.md](https://github.com/MetabolicAtlas/MetabolicAtlas/blob/develop/CONTRIBUTING.md). All ideas and contributions are highly appreciated.
 
+If you discover any potential vulnerabilities associated with the project, please reach out to us at [contact@metabolicatlas.org](mailto:contact@metabolicatlas.org)
```
Comment (MalinAhlberg) [inline README.md:16]:
```suggestion
If you discover any potential vulnerabilities associated with the project, please reach out to us at [contact@metabolicatlas.org](mailto:contact@metabolicatlas.org).
```

**Review by MalinAhlberg — APPROVED** [review]
Looks good !

File: README.md (hunk 1 above)
Comment (nanjiangshu) [reply README.md:16]: I think it would be better to use `reach out for us` according to https://idioms.thefreedictionary.com/reach+out+for+us

**Review by nanjiangshu — APPROVED** [review]
Looks great @e0. I just have a minor comment apart from what Malin suggested.

File: README.md (hunk 1 above)
Comment (e0) [reply README.md]: Thank you @MalinAhlberg !

@nanjiangshu I looked at the page you linked to and I think what we are using is correct based on their examples.

"If you would like to request a refund, please reach out to customer service."

vs.

"I reached out for my glass of water, ..."

**Comment by mihai-sysbio** [comment]
> * Update CII Badge <- This should be done after the PR is approved and merged.

Nice job! Just want to ping that the to-do is not checked, so maybe it hasn't been done yet?

**Comment by e0** [comment]
> > * Update CII Badge <- This should be done after the PR is approved and merged.
> 
> Nice job! Just want to ping that the to-do is not checked, so maybe it hasn't been done yet?

Nice catch, thanks! Updated now.

**Pull request merged**